File: KuroganeHammer.Web.Redux/app/domain/movement.ts
module app.domain {
    export interface IMovement extends ng.resource.IResource<IMovement>, Movement{
        
    }

    export class Movement {

        public id: number;
        public ownerId: number;

        constructor(
            public name: string,
            public value: number) {
        }
    }
}

File: FrannHammer.Domain/Movement.cs
using FrannHammer.Domain.Contracts;
using static FrannHammer.Domain.FriendlyNameMoveCommonConstants;
using static FrannHammer.Domain.FriendlyNameCommonConstants;

namespace FrannHammer.Domain
{
    public class Movement : MongoModel, IMovement
    {
        [FriendlyName(OwnerIdName)]
        public int OwnerId { get; set; }

        [FriendlyName("value")]
        public string Value { get; set; }

        [FriendlyName(OwnerName)]
        public string Owner { get; set; }
    }
}
